packages/ant-component-mapper/src/files/plain-text.js

```diff
@@ -4,8 +4,8 @@ import PropTypes from 'prop-types';
 
 const { Paragraph } = Typography;
 
-const PlainText = ({ label, name, ...rest }) => (
-  <Typography>
+const PlainText = ({ label, name, TypographyProps, ...rest }) => (
+  <Typography {...TypographyProps}>
     {label.split('\n').map((paragraph, index) => (
```

Comment:
^^ PlainText will crash if it has a Node type. Either modify this component it change the TS type ti string.